dashboard/15-final/app/lib/placeholder-data.js

```diff
@@ -1,4 +1,4 @@
-// This file contains placeholder data that you'll be replacing with real data in Chapter 7.
+// This file contains placeholder data that you'll be replacing with real data in Fetching Data chapter.
```

Comment:
Removing explicit chapter number here. @timeyoutakeit should they be lowercased though?

Reply:
I think this way is fine to be capitalized. Maybe worth adding a link too? i assume in most folk's editor it'll be clickable?

Reply:
Link added!